docs/design/2022-03-03-rc-read-tso-optimization.md

```diff
@@ -0,0 +1,36 @@
+# Read-Consistency Read With Tso Check
+
+- Author(s): cfzjywxk
+- Last updated: March. 3, 2022
+- Discussion at:
+
+## Motivation
+
+For the read-consistency isolation level read requests in a single transaction, each will need to fetch a new `tso` to read the latest committed data.
```

Comment:
read-consistency? is it read-committed?

Reply:
It actually refers to the term "statement-level read consistency" from Oracle. Yes, seems better to clarify it's the semantics of "read committed" in TiDB.